letsencrypt/run.sh

```diff
@@ -54,8 +61,12 @@ if [ ! -d "$CERT_DIR/live" ]; then
     done
 
     echo "$DOMAINS" > /data/domains.gen
-    if [ "$CHALLENGE" == "dns" ]; then
-        certbot certonly --non-interactive --config-dir "$CERT_DIR" --work-dir "$WORK_DIR" "--$DNS_PROVIDER" "--${DNS_PROVIDER}-credentials" "/data/dnsapikey" --email "$EMAIL" --agree-tos --config-dir "$CERT_DIR" --work-dir "$WORK_DIR" --preferred-challenges "$CHALLENGE" "${DOMAIN_ARR[@]}"
+    if bashio::config 'challenge' == "dns"; then
```

Comment:
This comparison would not work. Have you tested the add-on?

Reply:
Yes, but it slipped my eyes, as it was always true and i just tested against dns.
Fixed.